core/controllers/contributor_dashboard.py

```diff
@@ -196,9 +196,13 @@ def _get_skill_opportunities_with_corresponding_topic_name(
         """
         # We want to focus attention on lessons that are part of a classroom.
         # See issue #12221.
-        classroom_topic_ids = []
-        for classroom_dict in config_domain.CLASSROOM_PAGES_DATA.value:
-            classroom_topic_ids.extend(classroom_dict['topic_ids'])
+        classroom_topic_ids: List[str] = []
+        classrooms = classroom_config_services.get_all_classrooms()
+        all_classrooms_dict = [classroom.to_dict() for classroom in classrooms]
```

Comment:
Can we use the Classroom objects here rather than converting to dict? In general dicts should only be used at the boundaries of our interfaces.